metals/src/main/scala/scala/meta/internal/metals/codeactions/InlineValueCodeAction.scala

```diff
@@ -0,0 +1,157 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.Defn
+import scala.meta.Pat
+import scala.meta.Template
+import scala.meta.Term
+import scala.meta.Tree
+import scala.meta.internal.metals.Compilers
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.ServerCommands
+import scala.meta.internal.metals.clients.language.MetalsLanguageClient
+import scala.meta.internal.metals.codeactions.CodeAction
+import scala.meta.internal.parsing.Trees
+import scala.meta.io.AbsolutePath
+import scala.meta.pc.CancelToken
+
+import org.eclipse.{lsp4j => l}
+
+class InlineValueCodeAction(
+    trees: Trees,
+    compilers: Compilers,
+    languageClient: MetalsLanguageClient,
+) extends CodeAction {
+
+  type CommandData = l.TextDocumentPositionParams
+
+  override def command: Option[ActionCommand] =
+    Some(ServerCommands.InlineValue)
+
+  override def kind: String = l.CodeActionKind.RefactorInline
+
+  override def contribute(params: l.CodeActionParams, token: CancelToken)(
+      implicit ec: ExecutionContext
+  ): Future[Seq[l.CodeAction]] = Future {
+    val pathStr = params.getTextDocument.getUri
+    val path = pathStr.toAbsolutePath
+    val range = params.getRange()
+    val action =
+      for {
+        termName <- getTermNameForPos(path, range)
+        (defn, isDefn) <- hasDefinitionInScope(termName)
+        if (isLocal(
+          defn
+        ) || !isDefn) // either the value definition is local or action was executed on a value reference
```

Comment:
```suggestion
         /* either the value definition is local, which can be always inlined since it's scoped to the file
            or action was executed on a value reference so inlining just the reference will not break anything*/
        if (isLocal(defn) || !isDefn)
```
what do you think about this?